### ISSUE TITLE
Would it be stupid to write a guide on how to install dnscrypt-proxy on Android?

### ISSUE THREAD

**Issue by hunkjazz** [DNSCrypt/dnscrypt-proxy#1457]
## Subject

* [x] I'd like to contribute to the project (code, documentation, website, advocacy, integration, ...)

## Description

<details>
  <summary>I am a newbie and have spent the last few days searching how to install dnscrypt-proxy on Android. I tried some apps, but no success. What really helped me was these two guides: </summary>

- [Installing dnscrypt-proxy on Linux][1]
- [DNScrypt proxy for Android install][2]

[1]: https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Installation-linux
[2]: https://forum.xda-developers.com/showpost.php?s=bbb2ba547d7007611cafc666ba8f91b1&p=54677865&postcount=9

Although, these guides are either: a reference or broken. I had to adapt the installation method, by using these guides and other tips found around the web.

Now: this process was very stressfull for me, because I am a noob (like I said). I had to learn stuff and try others, before successfully installing dnscrypt-proxy.

Because of this, I was wondering if I should spend some hours writing and coding some stuff, in order to make other newbies lives easier.

**The contribution to the community would be:**

- A shell script wizard to help installing via ADB;
- A guide like [the one you guys did for Linux][1];
  - post it on a Gist;
  - post it on Android StackExchange;
  - post it on Xda Developers Forum.
- A guide on how to install via ADB manually;
  - post it on a Gist;
  - post it on Android StackExchange;
  - post it on Xda Developers Forum.
- A guide on how to install using file managers apps;
  - post it on a Gist;
  - post it on Android StackExchange;
  - post it on Xda Developers Forum.
- A guide on how to install using apps like AFWall (?);
  - post it on a Gist;
  - post it on Android StackExchange;
  - post it on Xda Developers Forum.
</details>

**tl;dr;**

I want to hoard and organize information found, in order to write some guides to help people installing dnscrypt-proxy on Android, easily.

Although, I am not sure if this is a good idea, because I'm affraid there is something similar/identical/better that I can't find.

**Comment by jedisct1** [comment]
Having a comprehensive Wiki page on that topic would definitely be very useful.

**Comment by realpanzer** [comment]
Just install Invizible Lite version and enable Dnscrypt module - protect DNS with Dnscrypt:
https://invizible.net/en/
Before you do that, go to fast settings and select Dnscrypt server (or add your own).
Click on start and you are done.
No root needed, works like a charm.
